integration/js/utils/WebdriverBrowserStack.js

```diff
@@ -70,11 +70,11 @@ const getBrowserStackOptions = (capabilities) => {
 
 const getBrowserStackUrl = () => {
   return (
-    'http://' +
+    'https://' +
     process.env.BROWSER_STACK_USERNAME +
     ':' +
     process.env.BROWSER_STACK_ACCESSKEY +
-    '@hub.browserstack.com/wd/hub'
+    '@hub-cloud.browserstack.com/wd/hub'
```

Comment:
Why did we change this one? Also, how did we test this?

Reply:
From PR testing information: For browserstack URL, I updated the URL based on steps mentioned when I created an account on BrowserStack for testing purpose, tested [Audio Integration Test](https://github.com/aws/amazon-chime-sdk-js/blob/master/integration/configs/audio_test.config.json) with my deployed browser meeting demo using the `browserstack` as the selenium grid provider with my username and access key. All runs passed in browserstack on Chrome and Firefox.

Reply:
Note: `hub` to `hub-cloud` change is done based on the documentation provided by the BrowserStack:
https://www.browserstack.com/docs/automate/selenium/getting-started/nodejs#run-your-first-test